src/stan-users-guide/simple_linear_regression/regression_efficient.stan

```diff
@@ -0,0 +1,19 @@
+data {
```

Comment:
This shouldn't be called "efficient" because it's really the baseline.  The more efficient thing to do is to use the glm function, which would have slightly different data types.